.azure-ci.yml

```diff
@@ -0,0 +1,46 @@
+# Disable triggers for now. TODO: add right merge triggers.
+trigger: none
```

Comment:
Are you using the dnceng build def to trigger Official builds?

Reply:
Official builds will not be per merge, they will be scheduled builds. I think there is no way to set scheduled builds from the YAML, so I plan on overriding triggers for official builds from the UI because of the complexity